Assets/HoloToolkit/Sharing/Prefabs/ManualIpConfigUtility.prefab

```diff
@@ -0,0 +1,3126 @@
+%YAML 1.1
```

Comment:
Personally, I'd prefer if this UI were separated from HoloToolkit.Sharing. I use Unity networking in my applications and would find this control useful if it wasn't tied to the HoloToolkit.Sharing library.